selector/methods/distance.py

```diff
@@ -60,28 +61,43 @@ class MaxMin(SelectionBase):
     Structure‐Activity Relationships 21.6 (2002): 598-604.
     """
 
-    def __init__(self, fun_dist=None):
+    def __init__(self, fun_dist=None, ref_index="medoid"):
         """
         Initializing class.
 
         Parameters
         ----------
         fun_distance : callable
             Function for calculating the pairwise distance between sample points.
-            `fun_dist(X) -> X_dist` takes a 2D feature array of shape (n_samples, n_features)
+            `fun_dist(x) -> x_dist` takes a 2D feature array of shape (n_samples, n_features)
             and returns a 2D distance array of shape (n_samples, n_samples).
+        ref_index: int, str, list, optional
+            Index of the reference sample to start the selection algorithm from.
+            It can be an integer, a float, a list of integers or floats or a mixture of them,
+            or "medoid". When "medoid", the medoid center is chosen as the reference
+            sample. Default is "None".
+
+        Notes
+        -----
+        When the `ref_index` is a list for multiple classes, it will be shared among all clusters.
+        If we want to use different reference indices for each class, we can perform the subset
+        selection for each class separately where different `ref_index` parameters can be used.
+        For example, if we have two classes, we can pass `ref_index=[0, 1]` to select samples from
+        class 0 and `ref_index=[3, 6]` class 1 respectively.
+
```

Comment:
It is best to add a summary of this note to the `ref_index` argument description.

Reply:
Fixed.